### ISSUE TITLE
MQTT Doorbell button

### ISSUE THREAD

**Issue by vmarks** [mrlt8/docker-wyze-bridge#168]
I would like the press of the doorbell button to send an MQTT message. I'm not certain that it does this currently from the documentation.

Is it possible?

**Comment by bobpaul** [comment]
>No expert here but I have an input boolean exposed to Alexa that reacts to the doorbell pressed. 

I think a way to do this without Alexa would be nice (someone using Apple Homekit, Google Home, or Home Assist without these 3 company's products).

**Comment by krysiswbk** [comment]
> > No expert here but I have an input boolean exposed to Alexa that reacts to the doorbell pressed.
> 
> I think a way to do this without Alexa would be nice (someone using Apple Homekit, Google Home, or Home Assist without these 3 company's products).

Agreed. Problem is Google doesn't allow trigger based routines. Does Homekit? If it does you could use a helper

**Comment by mrlt8** [comment]
This is tricky and similar to motion detection - wyze doesn't seem to have a way for the app to actually check the motion/button status, and all the motion/button data is probably being pushed over MQTT from the camera/doorbell to wyze over AWS IoT Core which then does the push notification to the phone/IFTTT/Alexa/etc.

There does seem to be a way to push the quick responses back to the doorbell with `k11635_responseQuickMessage`, but not sure if someone has to ring the doorbell for that to work.

If anyone want to look into the the wyze app, I believe you can find all the doorbell chime related stuff under `DDQ`.
